src/i18n/en.ts

```diff
@@ -244,6 +244,7 @@ export default {
             "Abbreviations are not allowed ('Value added tax' instead of 'VAT'). The label is written in sentence case " +
             "- first letter in upper case, the others in lower case. Label should not be changed, as its change might impact " +
             "meaning of data described by this concept.",
+        "term.label.confirmed.disabled": "The status of the term is CONFIRMED, so its label should not change.",
```

Comment:
Why is CONFIRMED in upper case (cf. the Czech variant)

Reply:
This belongs to #45 .